core/src/main/java/org/apache/calcite/avatica/BuiltInConnectionProperty.java

```diff
@@ -77,7 +77,10 @@
   TRUSTSTORE_PASSWORD("truststore_password", Type.STRING, null, false),
 
   HOSTNAME_VERIFICATION("hostname_verification", Type.ENUM, HostnameVerification.STRICT,
-      HostnameVerification.class, false);
+      HostnameVerification.class, false),
+
+  /** Fetch size limit, default is 100 rows. */
+  FETCH_SIZE("fetch_size", Type.NUMBER, AvaticaStatement.DEFAULT_FETCH_SIZE, false);
```

Comment:
I would recommend to name it `DEFAULT_FETCH_SIZE`

Reply:
I'm not clear on your suggestion.  Are you suggesting that the url connection parameter be named "DEFAULT_FETCH_SIZE" or as you suggest above "defaultFetchSize".  You mention having looked at a few other drivers I've looked too and don't see much consistency.  I chose "fetch_size" because that seemed consistent with the other parameters.  Would "default_fetch_size" be agreeable?

Reply:
yes, default_fetch_size for the jdbc property, which means the enum become DEFAULT_FETCH_SIZE (that was my point :) )